packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx

```diff
@@ -23,9 +24,17 @@ const LanguagesForm = ({ isManageLanguageVisible = false }: Props) => {
   const { watch, control, register } = useFormContext<SignInExperienceForm>();
   const isAutoDetect = watch('languageInfo.autoDetect');
   const [isManageLanguageFormOpen, setIsManageLanguageFormOpen] = useState(false);
+  const { languages } = useUiLanguages();
 
-  const { context: customPhrasesContext, Provider: CustomPhrasesContextProvider } =
-    useCustomPhrasesContext();
+  const languageOptions = useMemo(() => {
+    return languages.map((languageTag) => ({
+      value: languageTag,
+      title: uiLanguageNameMapping[languageTag],
+    }));
+  }, [languages]);
```

Comment:
I think this can be put in the `use-ui-languages` hook as well, and it's likely to be re-used in multiple places

Reply:
No, we encounter this option in the SIE config, preview, and the language management modal, they're different.